components/automate-backend-postgresql/habitat/plan.sh

```diff
@@ -1,10 +1,12 @@
 # shellcheck disable=SC2148
-UPSTREAM_PKG_IDENT="core/postgresql11/11.2"
+# stable channel
+
+UPSTREAM_PKG_IDENT="core/postgresql11/11.11"
```

Comment:
is it fine with A2HA team?

Reply:
@punitmundra  Please confirm.

Reply:
@vivek-yadav : we can go ahead with this changes. we will test it after merge this PR